new/LIT7150Miracle.xml

```diff
@@ -69,71 +69,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                        <desc>see <ref type="mss" corresp="EMML4205">EMML 4205, ff. 25v, 51r,
-                            26r</ref>
-                        </desc>
-                    </relation>
-                    <relation name="lawd:hasAttestation" active="LLIT7150Miracle" passive="EMML2378">
-                        <desc>
-                            <ref type="mss" corresp="EMML2378">EMML 2378, f. 31r</ref>
-                        </desc>
-                    </relation>
-                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="BNFabb114">
-                        <desc>
-                            <ref type="mss" corresp="BNFabb114"> BNFabb114, f. 18ra </ref>
-                        </desc>
-                    </relation>
-                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML2059">
-                        <desc>
-                            <ref type="mss" corresp="EMML2059">EMML 2059, f. 135r</ref>
-                        </desc>
-                    </relation>
-                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML2060">
-                        <desc>
-                            <ref type="mss" corresp="EMML2060">EMML 2060, f. 183r</ref>
-                        </desc>
-                    </relation>
-                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML2066">
-                        <desc>
-                            <ref type="mss" corresp="EMML2066">EMML 2066, f. 60r, 115v</ref>
-                        </desc>
-                    </relation>
-                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML2802">
-                        <desc>
-                            <ref type="mss" corresp="EMML2802">EMML 2803, f. 66v</ref>
-                        </desc>
-                    </relation>
-                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML6196">
-                        <desc>
-                            <ref type="mss" corresp="EMML6196">EMML 6196, f. 19v</ref>
-                        </desc>
-                    </relation>
-                    <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="EMML7543">
-                        <desc>
-                            <ref type="mss" corresp="EMML7543">EMML 7543, f. 27r</ref>
-                        </desc>
-                    </relation>
+                    <relation name="saws:formsPartOf" active="LIT7150Miracle" passive="LIT5673AkkonuColl"/>
+                    <relation name= "saws:isVersionOf" active="LIT7150Miracle" passive="LIT3592Miracle"/>
+                    <relation name="saws:isDifferentTo" active="LIT7150Miracle" passive="LIT7151iracle"/> 
                 </listRelation>
+                <listWit xml:id="all">
```

Comment:
listWit does not need xml:id and must go into sourceDesc above

Reply:
unless you refer to the xml:id of manuscripts in the list you do not need any of the xml:id assigned in the list below, please remove